src/platforms/apple/common/usage.mdx

```diff
@@ -52,5 +52,6 @@ This integration will swizzle some methods to create breadcrumbs e.g.: for `UIAp
 
 ## Known Limitations
 
+- Since version 6.0.0 of this SDK, Sentry on-premise version >= v20.6.0 is required. If you are using sentry.io, no action is needed.
```

Comment:
```suggestion
- Since version 6.0.0 of this SDK, Sentry on-premises version greater than version 20.6.0 is required. If you are using sentry.io, no action is needed.
```